packages/controller/src/Controller.ts

```diff
@@ -404,6 +408,34 @@ export default class Controller {
 		logger.info("Goodbye");
 	}
 
+	parseTrustedProxies() {
+		const trustedProxies = new BlockList();
+		const proxiesString = this.config.get("controller.trusted_proxies");
+		if (proxiesString) {
+			const proxies = proxiesString.split(",").map(s => s.trim());
```

Comment:
Thoughts on allowing the list to be split by spaces and commas, same as how they are allowed for port ranges?
Could use `.split(/[, ]/)` then filtering out empty strings after trim.

Reply:
The port ranges allowing space delimited values is more of a byproduct of making the parsing logic simpler to implement. IMHO it would be better to rework the port ranges parsing to be more strict.